packages/runtime/src/type/config.ts

```diff
@@ -73,7 +73,8 @@ export type Shared = {
   loading?: null | Promise<any>;
   // compatibility with previous shared
   eager?: boolean;
-  strategy: 'version-first' | 'loaded-first';
+  //TODO: need to implement types or another api hook
+  strategy: 'version-first' | 'loaded-first' | any;
```

Comment:
i think it needs to implement types :D

Reply:
ahh stratagy can be reversed not, i removed custom strategy as we have resolver now